### DISCUSSION TITLE
Welcome to go-qlc Discussions!

### DISCUSSION THREAD

**Discussion by gythialy** [qlcchain/go-qlc#1299] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ioannistsil** [comment]
hello i want to ask about qlc chain , roadmap and when billing platform will be ready for enterprice use, since allen li dont communicate with community we all in the dark waters!!

**Comment by ioannistsil** [comment]
anyone going to answer?
